lib/python/milc.py

```diff
@@ -17,6 +17,7 @@
 import logging
```

Comment:
don't feel comfortable reviewing this code as it uses a lot of python specific idioms